src/jsonapi-vuex.js

```diff
@@ -574,6 +575,15 @@ const normToStore = (record) => {
   return store
 }
 
+const processIncludedRecords = (context, results) => {
+  if ('included' in results.data) {
```

Comment:
`results` now won't always have a data property (i.e. `204` `No Content` responses), so this might fail. Probably need something more like:

`if (get(results, 'data', {}).hasOwnProperty('included')) {`